### PR TITLE
chore(table): buttons

### PR DESCRIPTION
**Pull request by mattnolting**
references: #5728

### CODE REVIEW

File: src/patternfly/components/Table/table.scss [hunk 1]
```diff
@@ -705,30 +683,16 @@
         --#{$table}__expandable-row--after--BorderColor: var(--#{$table}__tbody--m-clickable--m-expanded--BorderColor);
       }
 
-      &:hover,
-      &:focus,
-      &:active {
+      &:is(:hover, :focus) {
+        --#{$table}__tbody--m-clickable--BoxShadow: var(--#{$table}__tbody--m-clickable--hover--BoxShadow);
+        --#{$table}__tbody--m-clickable--BackgroundColor: var(--#{$table}__tbody--m-clickable--hover--BackgroundColor);
+
```
Comment (srambach) [inline src/patternfly/components/Table/table.scss:689]: If I'm not mistaken - can't all the BoxShadows be removed?

**Review by mcoker — APPROVED** [review]
🎃

**Review by srambach — COMMENTED** [review]
Pending all the discussion about buttons and alignment... just didn't know if box shadows should be removed in this PR..I don't think Figma shows any, unless I'm missing it.

File: src/patternfly/components/Table/table.scss (hunk 1 above)
Comment (srambach) [reply src/patternfly/components/Table/table.scss:689]: Next PR! 🚀

**Review by srambach — APPROVED** [review]

**Pull request merged**